documentation/src/docs/deployment/manual-install.md

```diff
@@ -19,27 +19,34 @@ Setup a HDFS storage cluster running **HDFS version 2.7+**. HDFS is used as Tier
 sufficient capacity to store the contents of all the streams. The storage cluster is recommended to be run
 alongside Pravega on separate nodes.
 
+### Filesystem
+
+HDFS is the default Tier 2 storage which is also called Long Term Storage. If it is easier to mount an nfs share, then Filesystem can be used in place of HDFS. The following configuration options are necessary to configure the Filesystem as Long Term Storage.
```

Comment:
1. Do not define Tier 2 <-> Long Term Storage here. There is another place in the doc where we should do this.
2. "nfs" -> "NFS".
3. "Filesystem" -> "FILESYSTEM"

Reply:
Yes Andrei